src/core/Akka.Tests/IO/TcpIntegrationSpec.cs

```diff
@@ -346,22 +346,22 @@ public void When_multiple_concurrent_writing_clients_All_acks_should_be_received
                 });
                 
                 // All acks should be received
-                clients.ForEach(client =>
+                await clients.ForEachAsync( async client =>
```

Comment:
Why not use a simple `foreach(var client in clients){ }`?
We don't need yet another useless utility method.